core/storage/app_feedback_report/gae_models.py

```diff
@@ -0,0 +1,591 @@
+                base_models.ID_LENGTH)
+            new_id = '%s.%s.%s' % (
+                datetime.datetime.utcnow().second, name_hash, random_hash)
+            if not cls.get_by_id(new_id):
+                return new_id
+        raise Exception(
+            'The id generator for AppFeedbackReportTicketModel is producing too'
+            'many collisions.')
+
+    @staticmethod
+    def get_deletion_policy():
+        """Model doesn't contain any information directly corresponding to a
+        user.
+        """
+        return base_models.DELETION_POLICY.NOT_APPLICABLE
+
+    @classmethod
+    def get_export_policy(cls):
+        """Model doesn't contain any data directly corresponding to a user."""
+        return dict(super(cls, cls).get_export_policy(), **{
+            'ticket_name': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'github_issue_number': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'archived': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'newest_report_timestamp': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'report_ids': base_models.EXPORT_POLICY.NOT_APPLICABLE
+        })
+
+    @staticmethod
+    def get_model_association_to_user():
+        """Model doesn't contain any user data."""
+        return base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
+
+    @staticmethod
+    def get_lowest_supported_role():
+        """The lowest supported role for feedback report tickets will be
+        moderator.
+        """
+        return feconf.ROLE_ID_MODERATOR
+
+
+class AppFeedbackReportStatsModel(base_models.BaseModel):
+    """Model for storing aggregate report stats on the tickets created.
+
+    Instances of this model contain statistics for different report types based
+    on the ticket they are assigned to and the date of the aggregation is on.
+
+    The id of each model instance is calculated by concatenating the platform,
+    ticket ID, and the date (in isoformat) this entity is tracking stats for.
+    """
+
+    # The unique ticket ID that this entity is aggregating for.
+    ticket_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The platform that these statistics are for.
+    platform = datastore_services.StringProperty(
+        required=True, indexed=True, choices=PLATFORM_CHOICES)
+    # The date in UTC that this entity is tracking on -- this should correspond
+    # to the creation date of the reports aggregated in this model.
+    stats_tracking_date = datastore_services.DateProperty(
+        required=True, indexed=True)
+    # JSON struct that maps the daily statistics for this ticket on the date
+    # specified in stats_tracking_date. The JSON will look contain two keys:
+    # daily_param_stats and daily_total_reports_submitted.
+    # The daily_param_stats will map each param_name (defined below
+    # by the const ALLOWED_STATS_PARAM_NAMES) to a dictionary of all the
```

Comment:
I don't see this defined anywhere--am I missing something?

Reply:
Ah this constant is defined in a later PR with the domain objects -- clarified in the comment but planning to add the specific values in the later PR.